browser/brave_drm_tab_helper.cc

```diff
@@ -5,14 +5,13 @@
 
 #include "brave/browser/brave_drm_tab_helper.h"
 
+#if BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT) || BUILDFLAG(BUNDLE_WIDEVINE_CDM)
```

Comment:
all of these guards seem a bit silly, why would we add this tab helper at all without them? I would put the guard on `BraveDrmTabHelper::CreateForWebContents(web_contents);` and also for the file in gn and the include in brave_tab_helpers

Reply:
I'm also a bit confused with these guards... @simonhong could you please clarify which values do they have on each platform?

Reply:
`brave_drm_tab_helper` files are guarded by gn `enable_widevine`.
`ENABLE_WIDEVINE_CDM_COMPONENT` is only used on Win and macOS and `BUNDLE_WIDEVINE_CDM` is only used on Linux.

Actually, we don't bundle it and download and install Widevine instead on linux during the runtime.
So, it doesn't make sense to use `BUNDLE_XXX` flags for use on linux also. (Chrome bundles it)
But, we reuse most of logic from `BUNDLE_WIDEVINE_CDM` on linux.
That's why `BUNDLE_WIDEVINE_CDM` flag is on our linux build.